index.html

```diff
@@ -1,96 +1,184 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-	<meta charset="utf-8">
-	<title>Home</title>
-	<link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
-	<link rel="stylesheet" href="css/index.css">
-	<!--[if IE]>
+    <meta charset="utf-8">
```

Comment:
What do you need to add to your `<head>` in order for your mobile styles to kick in?